client/post-editor/media-modal/index.jsx

```diff
@@ -127,6 +127,7 @@ export class EditorMediaModal extends Component {
 		return {
 			filter: '',
 			detailSelectedIndex: 0,
+			source: '',
```

Comment:
Although `source` is defined in the `getDefaultState` utility function, neither `source` or `onSourceChange` are defined as props or have default values.